ui/pages/confirm-transaction-base/confirm-transaction-base.component.js

```diff
@@ -981,6 +1085,7 @@ export default class ConfirmTransactionBase extends Component {
   componentWillUnmount() {
     this._beforeUnloadForGasPolling();
     this._removeBeforeUnload();
+    this._clearPollingInterval();
```

Comment:
@darkwing bump - just want to double-check this won't cause any memory leaks. If we're unsure, we can add the remove listener just in case.

```suggestion
    window.removeEventListener('beforeunload', () => this._clearPollingInterval);
    this._clearPollingInterval();
```

I see `this._removeBeforeUnload();` is being called above, but it's removing the event with a different callback
```
  _removeBeforeUnload = () => {
    window.removeEventListener('beforeunload', this._beforeUnloadForGasPolling);
  };
```